### ISSUE TITLE
Which pretrained model did you try to use in fcos.py ?

### ISSUE THREAD

**Issue by ajithvcoder** [yoyomimi/TNT_pytorch#5]
Is it https://dl.fbaipublicfiles.com/detectron/ImageNetPretrained/MSRA/R-50.pkl ? which was mentioned in detect.sh file

**Comment by yoyomimi** [comment]
Yes, I used that one, R-50.pkl.

**Comment by ajithvallabai** [comment]
Hi @yoyomimi  i tired to use the R-50.pkl file and train it on MOT-16 dataset but it is giving me "nan" error after 12 epochs .I have shared the below notebook .Could you tell any way to fix it 
https://colab.research.google.com/drive/1eztUFIiydEi_nBYR3aJtOQC8IB1fkvzC?usp=sharing

**Comment by ajithvallabai** [comment]
@yoyomimi also could you tell why track id is also a output of object detector .

**Comment by yoyomimi** [comment]
Sorry for that I missed your message for several days.
I've never met such problems. But you can check your whether your `num_gt ` in file fcos_loss.py is zero. (Actually there is related code in line 214 to deal with such condition...) Sometimes it may generate nan loss on detection tasks when the amount of gt is zero while there's no corresponding code.
Generally, we get nan when we try to devide sth by zero, check whether there is any possible denominator equals to zero.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Ajith"<notifications@github.com&gt;; 
发送时间: 2020年8月31日(星期一) 下午5:23
收件人: "yoyomimi/TNT_pytorch"<TNT_pytorch@noreply.github.com&gt;; 
抄送: "陈明霏"<fei912509706@qq.com&gt;; "Mention"<mention@noreply.github.com&gt;; 
主题: Re: [yoyomimi/TNT_pytorch] Which pretrained model did you try to use in fcos.py ? (#5)





 
Hi @yoyomimi  i tired to use the R-50.pkl file and train it on MOT-16 dataset but it is giving me "nan" error after 12 epochs .I have shared the below notebook .Could you tell any way to fix it
 https://colab.research.google.com/drive/1eztUFIiydEi_nBYR3aJtOQC8IB1fkvzC?usp=sharing

**Comment by ajithvcoder** [comment]
okay thanks
